### ISSUE TITLE
Determination for `kIsLineBreakingObject` a11y attribute

### ISSUE THREAD

**Issue by yaakovschectman** [flutter/flutter#118184]
To properly implement UIA for Windows a11y, we will need to determine when a node should have the `kIsLineBreakingObject` boolean attribute set. We do not currently use it. For an interim solution, I intend to set the attribute on all nodes by default.

**Comment by yaakovschectman** [comment]
Unassigning while UIA is low priority.
